scripts/gitlab/ci-build-test.sh

```diff
@@ -23,16 +23,16 @@ if [ "${CI_MACHINE}" == "lassen" ]; then
   LLVM_INSTALL_DIR=$(llvm-config --prefix)
   CMAKE_OPTIONS_MACHINE=" -DENABLE_CUDA=on -DCMAKE_CUDA_ARCHITECTURES=70 -DCMAKE_CUDA_COMPILER=$LLVM_INSTALL_DIR/bin/clang++"
 elif [ "${CI_MACHINE}" == "tioga" ]; then
-  ml load rocm/5.7.1
+  ml load rocm/${PROTEUS_CI_ROCM_VERSION}
 
-  LLVM_INSTALL_DIR=/opt/rocm-5.7.1/llvm
+  LLVM_INSTALL_DIR=${ROCM_PATH}/llvm
```

Comment:
Check if ${ROCM_PATH} is set and print the actual value. Can be beneficial to debug issues at some point.